java/src/test/java/org/certificatetransparency/ctlog/LogSignatureVerifierTest.java

```diff
@@ -47,6 +20,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.certificatetransparency.ctlog.TestData.*;
```

Comment:
Please don't change the order of the imports, nor use wildcard imports.

https://google.github.io/styleguide/javaguide.html#s3.3-import-statements